### ISSUE TITLE
[1.20.1] Fried Egg Doesn't Show Up As A Recipe for Incubation or Farmer's Delight If Installed Together

### ISSUE THREAD

**Issue by RegalRuffRaff** [team-abnormals/incubation#30]
Incubation disables the Farmer's Delight Fried Egg recipe. Applies to 4.0.2 and 4.0.0 for 1.20.1. Additionally, Incubation's own Fried Egg doesn't show up in JEI or as a recipe in 4.0.0+ with Farmer's Delight installed. All Fried Egg types cannot be cooked.

Tested with:
Farmer's Delight 1.2.4
Incubation 4.0.2 + Incubation 4.0.0

**Comment by WordStormRose** [comment]
I am also having this issue.

**Comment by bageldotjpg** [comment]
Just tested and is working fine for me... If abnormals delight is not present both recipes load, one will take priority (not sure which), abnormals delight disables FD's

**Comment by RegalRuffRaff** [comment]
Alright, I guess I'll not use Abnormals Delight. Thanks for letting me know !

**Comment by bageldotjpg** [comment]
Abnormals Delight disables FDs so there isn't a race condition determining which one is enabled. I'd recommend keeping it. If you really need the FD egg over the incubation one for whatever reason you could make a datapack but functionally the incubation and FD ones are basically the same besides the texture

**Comment by RegalRuffRaff** [comment]
Right, so after trying it out with 4.0.4, it seems to be working just fine. I have no preference for either fried egg tbh. Maybe there was some other mod that was messing with something. I posted this issue so long ago that I don't remember the exact list I had at the time, but everything seems fine now. I'll keep Abnormals Delight !
